site/themes/s2b_hugo_theme/static/legacycaljs/addevent.js

```diff
@@ -236,7 +238,7 @@
             var date = $form.formatDate(value['date']);
             var displayDate = $form.formatDate(value['date'], abbreviated=true);
             var newsflash = value['newsflash'];
-            var cancelled = value['status'] === 'C';
+            var cancelled = (value['status'] === 'C');
```

Comment:
Added parenthesis to make it more obvious (for me, at least) that it's a computed true/false value.